pkg/network/netbinding/netbinding.go

```diff
@@ -48,6 +50,23 @@ func NetBindingPluginSidecarList(vmi *v1.VirtualMachineInstance, config *v1.Kube
 	return pluginSidecars, nil
 }
 
+func warnUnregisteredPlugins(vmi *v1.VirtualMachineInstance, config *v1.KubeVirtConfiguration, recorder k8srecord.EventRecorder) {
+	var plugins map[string]v1.InterfaceBindingPlugin
+	if config.NetworkConfiguration != nil {
+		plugins = config.NetworkConfiguration.Binding
+	}
+
+	for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
+		if iface.Binding != nil {
+			if _, exist := plugins[iface.Binding.Name]; !exist {
+				msg := fmt.Sprintf("Network interface %[1]q is set with unregistered network binding plugin %[2]q. "+
+					"Make use the network binding plugin %[2]q is registered in Kubevirt config.", iface.Name, iface.Binding.Name)
+				recorder.Eventf(vmi, k8scorev1.EventTypeWarning, UnregisteredNetworkBindingPluginReason, msg)
```

Comment:
I think that the message can be built once per all the unregistered bindings, then sent once instead of one per interface.

Reply:
Done